lib/tjbot.js

```diff
@@ -806,6 +806,8 @@ TJBot.prototype.read = function() {
  * Returns the photo data in a Buffer.
  */
 TJBot.prototype._captureImage = function(filePath) {
+    this._assertCapability('see');
```

Comment:
This has been moved to the takePhoto() method.
Something important to note ... you use the node-raspistill library to take images. From my tests it takes **6.2** seconds on the average to take a picture. This is compared to the **1.1** seconds it takes with the previous spawn approach I used earlier. 6 seconds is large, is there any other reason we should go with node-raspistill?